eth_tester/backends/pyevm/main.py

```diff
@@ -111,6 +113,17 @@ def get_default_account_keys(quantity=None):
         yield private_key
 
 
+@to_tuple
+def get_account_keys_from_mnemonic(mnemonic, quantity=None):
```

Comment:
nit - could this be:
```suggestion
def get_account_keys_from_mnemonic(mnemonic, quantity=10):
```
And then we could get rid of line 119?

Reply:
Yes, I was copying the method defined above it; I can change them both since they are effectively the same thing

Reply:
Ah when running the tests, I see why they did it this way... It is because sometimes this is kwarg passthrough in which case `None` gets passed in. So the line `quantity = quantity or 10` handles the case when someone does `quantity=None`

Reply:
Ah, interesting. I think I slightly prefer leaving it as None then, but I'll leave it up to you!

Reply:
I change it back to `None` !